javascript/sbp.bundle.js

```diff
@@ -15817,65 +15915,97 @@ for (var i = 0, len = code.length; i < len; ++i) {
 revLookup['-'.charCodeAt(0)] = 62
 revLookup['_'.charCodeAt(0)] = 63
 
-function placeHoldersCount (b64) {
+function getLens (b64) {
```

Comment:
I wonder what are these changes, starting from here until the end of this file?
They don't seem to be related to the new message.